packages/react/src/components/Forms/Radio/index.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { Radio, RadioGroup } from "components/Forms/Radio/index";
+
+export default {
+  title: "Components/Forms/Radio",
+  component: Radio,
+} as ComponentMeta<typeof Radio>;
+
+const Template: ComponentStory<typeof Radio> = (args) => <Radio {...args} />;
```

Comment:
You should provide an aria-label in case Radio has no label to prevent a11y errors.
```suggestion
const Template: ComponentStory<typeof Radio> = (args) => <Radio {...args} aria-label="Radio" />;
```